cmd/object-handlers.go

```diff
@@ -173,24 +173,31 @@ func (api objectAPIHandlers) GetObjectHandler(w http.ResponseWriter, r *http.Req
 
 	setObjectHeaders(w, objInfo, hrange)
 	setHeadGetRespHeaders(w, r.URL.Query())
-	httpWriter := ioutil.WriteOnClose(writer)
 
 	getObject := objectAPI.GetObject
 	if api.CacheAPI() != nil && !hasSSECustomerHeader(r.Header) {
 		getObject = api.CacheAPI().GetObject
 	}
 
+	statusCodeWritten := false
+	httpWriter := ioutil.WriteOnClose(writer)
+
+	if hrange != nil && hrange.offsetBegin > -1 {
+		statusCodeWritten = true
+		w.WriteHeader(http.StatusPartialContent)
```

Comment:
for a Get with range header, this throws multiple response.WriteHeader calls warning.

Reply:
> for a Get with range header, this throws multiple response.WriteHeader calls warning.

oups yes, fixed.